File: VB/Custom Radial Menu/RadialMenuDescendant.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports DevExpress.XtraBars.Ribbon
Imports System.ComponentModel

Namespace WindowsFormsApplication4
    <ToolboxItem(True)> _
    Public Class MyRadialMenu
        Inherits RadialMenu

        Public Sub New()
        End Sub
        Protected Overrides Function CreateLayeredWindow() As RadialMenuWindow
            Return New MyRadialMenuWindow(Me)
        End Function
    End Class
End Namespace

File: CS/Custom Radial Menu/RadialMenuDescendant.cs
using System;
using System.Collections.Generic;
using System.Linq;
using DevExpress.XtraBars.Ribbon;
using System.ComponentModel;

namespace WindowsFormsApplication4 {
    [ToolboxItem(true)]
    public class MyRadialMenu : RadialMenu {
        public MyRadialMenu() {
        }
        protected override RadialMenuWindow CreateLayeredWindow() {
            return new MyRadialMenuWindow(this);
        }
    }
}
